web/modules/custom/do_ai_alt_text/src/Plugin/Action/RegenerateImageAltText.php

```
@@ -0,0 +1,200 @@
+
+    // A batch outlives the request that queued it, so access is re-checked
+    // rather than trusted from the form submission.
+    if (!$media instanceof MediaInterface || !$action->access($media)) {
+      $context['results'][self::RESULT_SKIPPED]++;
+
+      return;
+    }
+
+    $context['message'] = new TranslatableMarkup('Re-generating alt text for @label', ['@label' => $media->label()]);
+    $context['results'][$action->regenerate($media)]++;
+  }
```

PHPMD (2.15.0)

[error] The parameter $media_id is not named in camelCase. (undefined)

(CamelCaseParameterName)